src/frontend/src/components/OeciLogin/index.tsx

```diff
@@ -142,46 +143,62 @@ export default class OeciLogin extends React.Component<State> {
                     Log in to OECI
                   </button>
                 </fieldset>
-                <div className="black-70" role="alert">
-                  {this.state.missingInputs === true ? (
-                    <p id="inputs_msg" className="bg-washed-red mb3 pa3 br3 fw6">
-                      All fields are required.
-                    </p>
-                  ) : null}
-                  {this.state.expectedFailure === true ? (
-                    <p className="bg-washed-red mb3 pa3 br3 fw6">
-                      {this.state.expectedFailureMessage}
-                    </p>
-                  ) : null}
-                  {this.state.invalidResponse === true ? (
-                    <p
-                      id="no_match_msg"
-                      className="bg-washed-red mv4 pa3 br3 fw6"
+                <InvalidInput
+                  message="All fields are required."
+                  condition={this.state.missingInputs}
+                />
+                <InvalidInput
+                  message={""}
+                  condition={this.state.invalidResponse}
+                >
+                  <>
+                    We're experiencing technical difficulties, please contact{" "}
+                    <a
+                      className="link underline hover-blue"
+                      href="mailto:help@recordsponge.com"
                     >
-                      We're experiencing technical difficulties, please contact{' '}
-                      <a
-                        className="link underline hover-blue"
-                        href="mailto:help@recordsponge.com"
-                      >
-                        help@recordsponge.com
-                      </a>
-                    </p>
-                  ) : null}
-                </div>
+                      help@recordsponge.com
+                    </a>
+                  </>
+                </InvalidInput>
+                <InvalidInput
+                  message={this.state.expectedFailureMessage}
+                  condition={this.state.expectedFailure}
+                />
                 <p className="lh-copy mt4">
```

Comment:
I think we can change this `mt4` to `mt5` and remove the `mb4` from the log in button, and that should make the spacing look decent.